src/ProjectNeon/Assets/Features/Effects/DamageApplied.cs

```diff
@@ -0,0 +1,14 @@
+﻿public sealed class DamageApplied : Effect
+{
+    private Damage _damage;
+
+    public DamageApplied(Damage damage)
+    {
+        _damage = damage;
+    }
+
+    public void Apply(Member source, Target target)
+    {
+        target.Members[0].State.ChangeHp(-_damage.Calculate(source, target) * target.Members[0].State.Damagability());
```

Comment:
This members indexing hack is getting insane.

We can't add more bugs to the system.

No more indexing into Target.

Reply:
@SilasReinagel It will be fixed in #460

Reply:
@SilasReinagel It will be fixed in #460